### ISSUE TITLE
Order of operands not correct for follow-up calculations

### ISSUE THREAD

**Issue by Seeraeuber** [microsoft/calculator#1267]
**Describe the bug**

**Steps To Reproduce**

1. Do an initial calculation, e.g. 10*10 and hit Enter to get the result
2. Try to add (+) another multiplication, e.g. +3*5 and hit Enter
3. Calculator first executes the addition, and then multiplies --> Result is 515
![order_of_operands_2](https://user-images.githubusercontent.com/41448641/84481207-0bb98180-ac96-11ea-80cd-6607f9e84cae.png)

**Expected behavior**
Multiplication should happen before addition, expected result should be 115 instead of 515

**Screenshots**
![order_of_operands](https://user-images.githubusercontent.com/41448641/84476183-1112ce00-ac8e-11ea-99b0-99fa1dff5623.png)

**Device and Application Information**
 - OS Build: Win 10 Pro Version 10.0.18363 Build 18363
 - Architecture: X64
 - Application Version: 10.1910.0.0
 - Region: de-DE
 - Dev Version Installed: False

**Additional context**

**Requested Assignment**

I'm just reporting this problem.  I don't want to fix it.

**Comment by ItzLevvie** [comment]
Duplicate of https://github.com/microsoft/calculator/issues/138.

**Comment by astronsp** [comment]
Can I work on this problem

**Comment by Seeraeuber** [comment]
Well, interesting that the scientific calculator behaves different... That should fix my needs in the mean time. Thanks for pointing it out - i'll close this issue then :)

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by irevoire** [comment]
Well, actually this is still broken, maybe it would be good to fix it for the standard calculator too?

**Comment by grochocki** [comment]
#138 covers improving clarity of math expressions in history for Standard Calculator.
